devcareer/code-jammers-backend
# #174566485 Set up CI and CD and add badges to readMe

### What does this PR do?

This PR sets up continuous integration and deployment of the codebase using Code climate and Github actions.

### Description of tasks to be completed?

-  Add the repo to Code Climate
-  Set up Github actions for continuous integration
-  Add badges to the readMe file
   
### How should this be manually tested/checked?

- Check the `Action` tab on the repo: `https://github.com/devcareer/code-jammers-backend` to see build result
- Check the `readme` of the branch `ch-setup-ci-cd-174566485` to see the badges
    
### What are the relevant pivotal tracker stories?
[#174566485 ](https://www.pivotaltracker.com/story/show/174566485)
